app/pages/team.tsx

```diff
@@ -1,29 +1,37 @@
 import { Button } from '../frontend/components/button/button'
+import { useUsersQuery } from '../frontend/generated/graphql'
 
 const Team = (): JSX.Element => {
+  const [{ data, error }] = useUsersQuery()
   return (
     <article>
       <h2 className="flex justify-between">
-        <span>Your projects</span>
+        <span>Your Team</span>
       </h2>
       <table className="w-full">
         <thead>
           <tr>
+            <th />
             <th>Username</th>
             <th>Email</th>
             <th>Projects</th>
             <th />
           </tr>
         </thead>
         <tbody>
-          <tr>
-            <td>Linus</td>
-            <td>linus@xyz.de</td>
-            <td>Projekt 1, Projekt 2</td>
-            <td>
-              <Button variant="primarySlim">Details</Button>
-            </td>
-          </tr>
+          {data?.users.map((user) => (
+            <tr key={user.name}>
+              <td>
+                <img style={{ width: '20px' }} src={user.image ?? undefined} />
```

Comment:
```suggestion
                <img className="w-5" src={user.image ?? undefined} />
```

We should use tailwind for styles